File: PhoneStoreBackend/Enums/OrderStatusEnum.cs
namespace PhoneStoreBackend.Enums
{
    public enum OrderStatusEnum
    {
        ready_to_pick,        // Mới tạo đơn
        picking,     // Đang lấy hàng
        picked,        // Đã lấy hàng
        sorting,        // Hàng đang ở kho
        delivering,     // Đang giao
        delivered,      // Đã nhận hàng thành công
        delivery_fail,     // Giao hàng thất bại
        cancel,         // đã hủy
        pending,        // chờ xác nhận
    }
}

File: PhoneStoreFrontend/src/enums/OrderStatusEnum.ts
enum OrderStatusEnum {
  pending = 'Chờ xác nhận',
  ready_to_pick = 'Đã xác nhận',
  picking = 'Đang lấy hàng',
  picked = 'Đã lấy hàng',
  sorting = 'Hàng đang ở kho',
  delivering = 'Đang vận chuyển',
  delivered = 'Đã giao hàng',
  delivery_fail = 'Giao hàng thất bại',
  cancel = 'Đã hủy'
}

export default OrderStatusEnum
